src/components/Comment/index.tsx

```diff
@@ -40,9 +42,20 @@ const Comment: FC<PropsType> = ({ type, data }) => {
     setIsMore((prev) => !prev);
   }, []);
 
+  const onProfileClick = () => {
+    if (writerType === "admin" && "선생님") {
```

Comment:
```suggestion
    if (writerType === "admin") {
```
여기에 "선생님" 이라는 조건은 항상 참이기 때문에 필요없을거 같네요

Reply:
옙